### PR TITLE
Testing: 5 Case fixtures for testing callbacks

### PR DESCRIPTION
**Pull request by Pheonnexx**
## What does this pull request do?

add new fixtures for 5 cases for testing callbacks

## Any other changes that would benefit highlighting?

Intentionally left blank.

## Checklist

- [ ] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

**Comment by said-moj** [comment]
What do you think of also adding the loading of these end-to-end fixtures to [create_db.sh](https://github.com/ministryofjustice/cla_backend/blob/master/bin/create_db.sh)  maybe hid behind an environment variable like `LOAD_END_TO_END_FIXTURES`?

**Comment by jogilder** [comment]
> What do you think of also adding the loading of these end-to-end fixtures to [create_db.sh](https://github.com/ministryofjustice/cla_backend/blob/master/bin/create_db.sh) maybe hid behind an environment variable like `LOAD_END_TO_END_FIXTURES`?

I added my fixtures as a separate line in end to end tests 
https://github.com/ministryofjustice/cla-end-to-end-tests/blob/b8a9da9c40b0029d01dd721a6bc3d40687aeec73/behave/run_test_local.sh 
and also in the orb 
https://github.com/ministryofjustice/cla-end-to-end-tests/blob/159783350747e78c0557a4c3557d0fe70ff14bb0/orb.yml
Think would be better done as described by @said-moj. The other test fixtures should be moved as well.

File: helm_deploy/cla-backend/values-uat-static.yaml [hunk 1]
```diff
@@ -22,6 +22,8 @@ envVars:
       optional: true
   LOAD_TEST_DATA:
     value: "True"
+  LOAD_END_TO_END_FIXTURES:
+    value: "True"
```
Comment (said-moj) [inline helm_deploy/cla-backend/values-uat-static.yaml:26]: I don't think we want the end-to-end fixtures loaded on static uat

**Review by jogilder — APPROVED** [review]
LGTM

**Review by jogilder — APPROVED** [review]
Looks good, assuming we would always want end to end test data on dev environment.

**Pull request merged**